### PR TITLE
Add 12 icons

### PR DESCRIPTION
**Pull request by peterge1998**
## Description

Fixes #(issue)

## Type of change

:x: Bug fix (non-breaking change which fixes an issue)
:white_check_mark:  Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:x: General change (non-breaking change that doesn't fit the above categories like copyediting)

### Icons added
com.wizards.mtga
com.wizards.winter_orb
com.blizzard.wtcg.hearthstone
com.humble.SlayTheSpire
com.Afterburn.Railbound
de.human_resources
com.comdirect.phototan
com.github.premnirmal.tickerwidget
com.wireguard.android
com.commit451.gitlab
be.martinelli.miniflutt
com.ucdevs.jcross
com.lidl.eci.lidlplus
com.mattermost.rnbeta
com.valuephone.vpnetto
tipster.evolutioit.app

added link:
de.mcdonalds.mcdonaldsinfoapp

Reworked:
de.rki.covpass.app

### CODE REVIEW

File: app/src/main/res/xml/grayscale_icon_map.xml [hunk 1]
```diff
@@ -939,6 +949,7 @@
     <icon drawable="@drawable/steam_link" package="com.valvesoftware.steamlink" name="Steam Link" />
     <icon drawable="@drawable/stellio" package="io.stellio.music" name="Stellio" />
     <icon drawable="@drawable/stocard" package="de.stocard.stocard" name="Stocard" />
+    <icon drawable="@drawable/stock_widget" package="com.github.premnirmal.tickerwidget" name="Stock Widget" />
```
Comment (daywalk3r666) [inline app/src/main/res/xml/grayscale_icon_map.xml:952]: <icon drawable="@drawable/stock_ticker" package="com.github.premnirmal.tickerwidget" name="Stock Widget" />

**Review by daywalk3r666 — CHANGES_REQUESTED** [review]

**Comment by peterge1998** [comment]
In my opinion the design of magic & companion turned out fcking awesome.
I hope hearthstone & wireguard are fine for you, i just wanted to keep the details of the original logos...

**Comment by peterge1998** [comment]
and this wont be my last pr, mydealz & covpass are still waiting ;)

**Review by Chefski — CHANGES_REQUESTED** [review]
Could you make miniflutt.svg have a stroke of 12px?

Hearthstone has sharp corners which don't necassirily match lawnicons style.

Strokes need to be rounded to 8px or 12px:
- wireguard.svg
- slay_the_spire.svg
- railbound.svg (can be 6px)
- nonogramm_katana.svg (hwoever it would also need a redesign as some gaps are too small)
- magic_companion.svg (also needs to be recentered)
- labcoat.svg

As for the icons that have been rendered wrong, I'd just ask to remove them or try to get them fixed

**Comment by peterge1998** [comment]
I have already updated hearthstone, wg, miniflutt and slay t s to 8px & round corners.

**Review by Chefski — CHANGES_REQUESTED** [review]
Don't really know how I feel about the heartstone icon. The corners are not rounded in the main shape in the middle and has lots of paths. You can either try improve on it or remove it from the PR and maybe add it in the future.

Also miniflutt has strokes that are not whole numbers. Could you fix that?

**Comment by peterge1998** [comment]
> Don't really know how I feel about the heartstone icon.

Nah, i would like to add it now. Can we agree on its current state?

> Also miniflutt has strokes that are not whole numbers. Could you fix that?

Now it should be 12

**Comment by peterge1998** [comment]
> * railbound.svg (can be 6px)

Done!

> * nonogramm_katana.svg (hwoever it would also need a redesign as some gaps are too small)

Which gaps do you mean? The squares (at 1 3 and 1 2)? 

> * magic_companion.svg (also needs to be recentered)

The C is aligned vertical to the page, which is correct in my opinion:
image

> * labcoat.svg

**Comment by peterge1998** [comment]
👀

**Review by Chefski — CHANGES_REQUESTED** [review]
Here are the screenshots for the changes needed
image

image


Icons that new strokes with whole numbers:

- tipster
- lidl_plus

Icons needing resizing to 160x160:

- netto

**Comment by peterge1998** [comment]
Done :)
And I leave a small note here for my XML problem in android studio: If this occurs, go into Inkscape and convert the object not generating successful to a path (https://logosbynick.com/convert-text-to-a-path-in-inkscape/)

**Review by Chefski — APPROVED** [review]

**Comment by Chefski** [comment]
Thanks

**Pull request merged**